pydantic_extra_types/isbn.py

```diff
@@ -67,27 +64,50 @@ class Book(BaseModel):
 
     @classmethod
     def __get_pydantic_core_schema__(cls, source: type[Any], handler: GetCoreSchemaHandler) -> core_schema.CoreSchema:
+        """
+        Return a Pydantic CoreSchema with the ISBN validation.
+
+        Args:
+            source: The source type to be converted.
+            handler: The handler to get the CoreSchema.
+
+        Returns:
+            A Pydantic CoreSchema with the ISBN validation.
+
+        """
         return core_schema.with_info_before_validator_function(
             cls._validate,
             core_schema.str_schema(),
         )
 
     @classmethod
     def _validate(cls, __input_value: str, _: Any) -> str:
+        """
+        Validate a ISBN from the provided str value.
+
+        Args:
+            __input_value: The str value to be validated.
+            _: The source type to be converted.
+
+        Returns:
+            The validated ISBN.
+
+        Raises:
+            PydanticCustomError: If the ISBN is not valid.
+        """
         cls.validate_isbn_format(__input_value)
 
         return cls.convert_isbn10_to_isbn13(__input_value)
 
     @staticmethod
     def validate_isbn_format(value: str) -> None:
-        """
-        Validate a ISBN format from the provided str value.
+        """Validate a ISBN format from the provided str value.
 
         Args:
-            value: The str value representing the ISBN in 10 or 13 digits.
```

Comment:
I feel like this is a better description than the one that you've replaced it with...